src/main/java/de/internetsicherheit/brl/bloxberg/cache/ethereum/BloxbergClient.java

```diff
@@ -6,16 +6,17 @@
 import org.web3j.protocol.core.methods.response.EthBlock;
 import org.web3j.protocol.core.methods.response.EthBlockNumber;
 import org.web3j.protocol.core.methods.response.EthGetBlockTransactionCountByNumber;
+import org.web3j.protocol.core.methods.response.EthGetCode;
 import org.web3j.protocol.http.HttpService;
 
 import java.io.IOException;
 import java.math.BigInteger;
+import java.util.List;
 
 public class BloxbergClient {
 
     private final Web3j web3j;
-    private EthBlock rawBlock;
-    private BlockWithData blockWithData;
+    private Block blockWithData;
```

Comment:
Actually the class should not be stateful with regards to blockchain data. So this field should be removed. This was of course already an error in the previous design, but we should fix this while we are at it.